esmvaltool/cmorizers/data/downloaders/datasets/ndp.py

```diff
@@ -0,0 +1,40 @@
+"""Script to download APHRO-MA from its webpage."""
+import logging
+import os
+
+from esmvaltool.cmorizers.data.downloaders.wget import WGetDownloader
+from esmvaltool.cmorizers.data.utilities import unpack_files_in_folder
+
+logger = logging.getLogger(__name__)
+
+
+# pylint: disable=invalid-name
+def download_dataset(config, dataset, dataset_info, _, __, overwrite):
+    """Download dataset.
+
+    Parameters
+    ----------
+    config : dict
+        ESMValTool's user configuration
+    dataset : str
+        Name of the dataset
+    start_date : datetime
+        Start of the interval to download
+    end_date : datetime
+        End of the interval to download
+    overwrite : bool
+        Overwrite already downloaded files
+    """
+    downloader = WGetDownloader(
+        config=config,
+        dataset=dataset,
+        dataset_info=dataset_info,
+        overwrite=overwrite,
+    )
+
+    downloader.download_file(
```

Comment:
This download does not work for me. Could you please check? I'm not sure what kind of registration is really needed here because the file can be easily downloaded with the web browser without any authentification.

Reply:
It is fixed now. I needed to create the folder manually